.github/CODEOWNERS

```
@@ -43,15 +43,12 @@ xptifw/ @intel/llvm-reviewers-runtime
 
 # DPC++ tools
 llvm/ @intel/dpcpp-tools-reviewers
-llvm/include/llvm/SYCLLowerIR/MutatePrintfAddrspace.h @intel/dpcpp-tools-reviewers
-llvm/lib/SYCLLowerIR/MutatePrintfAddrspace.cpp @intel/dpcpp-tools-reviewers
-llvm/test/SYCLLowerIR/printf_addrspace/ @intel/dpcpp-tools-reviewers
 
 # Clang offload tools
 clang/tools/clang-offload-*/ @intel/dpcpp-tools-reviewers
 
 # Explicit SIMD
-SYCLLowerIR/ @intel/dpcpp-esimd-reviewers
+SYCLLowerIR/ESIMD @intel/dpcpp-esimd-reviewers
```

```suggestion
ESIMD/ @intel/dpcpp-esimd-reviewers
```
Will do the same + match other ESIMD directories anywhere in the repository.
I don't recall if gitignore rules are case sensitive or not, so it's possible that next rule is enough and we can just remove this one.

---

[Git config documentation](https://git-scm.com/docs/git-config#Documentation/git-config.txt-coreignoreCase) seems to suggest that case-sensitivity is on by default. I'll apply the current suggestion.